ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
So the `classes` option would stay empty by default, but we add `_addClass` calls with `ui-droppable-active` and `ui-droppable-hover` as the structural classes?